src/sentry/reprocessing.py

```diff
@@ -39,12 +39,21 @@ def get_reprocessing_revision(project, cached=True):
         pass
 
 
-def bump_reprocessing_revision(project):
+def bump_reprocessing_revision(project, use_buffer=False):
     """Bumps the reprocessing revision."""
+    from sentry import buffer
     from sentry.models import ProjectOption
 
     rev = uuid.uuid4().hex
-    ProjectOption.objects.set_value(project, REPROCESSING_OPTION, rev)
+    if use_buffer:
+        buffer.incr(
+            ProjectOption,
+            columns={},
+            filters={"project": project, "key": REPROCESSING_OPTION},
+            signal_only=True,
```

Comment:
`signal_only=True` so the actual update happens in receiver and will properly reload cache via the `ProjectOption.objects.set_value`.

Reply:
I absolutely love how you have to specify a model cls that, with signal_only, is only used as input for the redis key and nothing else.